api/src/pcapi/routes/backoffice/offers/forms.py

```diff
@@ -458,7 +458,10 @@ def __init__(self, *args: list, **kwargs: dict):
     )
     show_type = fields.PCSelectMultipleField(
         "Type de spectacle",
-        choices=[(str(s), show_types.SHOW_TYPES_LABEL_BY_CODE[s]) for s in show_types.SHOW_TYPES_LABEL_BY_CODE],
+        choices=[
+            (show_types.SHOW_TYPES_LABEL_BY_CODE[s], show_types.SHOW_TYPES_LABEL_BY_CODE[s])
+            for s in show_types.SHOW_TYPES_LABEL_BY_CODE
```

Comment:
Ça doit revenir au même :
```suggestion
            (label, label) for label in show_types.SHOW_TYPES_LABEL_BY_CODE.values()
```